content/en/tools/migrations.md

```diff
@@ -50,6 +50,7 @@ Alternatively, you can use the new [Jekyll import command](/commands/hugo_import
 - [exitwp-for-hugo](https://github.com/wooni005/exitwp-for-hugo) - A python script which works with the xml export from WordPress and converts WordPress pages and posts to Markdown and YAML for hugo.
```

Comment:
```suggestion
```

Reply:
I've made the suggested update and also updated the wpxr-to-static text, as it mentioned exitwp-to-hugo and said 'see above'.